assets/wizards/popups/components/segment-group/index.js

```diff
@@ -110,7 +112,14 @@ const SegmentGroup = props => {
 							<Button
 								isQuaternary
 								isSmall
-								onClick={ () => showPreview() }
+								onClick={ () => {
+									cookies.remove( 'newspack-cid' );
```

Comment:
This isn't working for me, because the `domain` and `path` attributes set for this cookie don't match the `js-cookie` defaults. It works if I change it like so:

```
cookies.remove( 'newspack-cid', {
	domain: '.' + window.location.host,
	path: '/',
} );
cookies.set( 'newspack-cid', `preview-${ Math.round( Math.random() * 9999 ) }`, {
	domain: '.' + window.location.host,
	path: '/',
} );

console.log( cookies.get( 'newspack-cid' ) );
```

I think more testing is needed to ensure that these attributes remain constant across environments, though.

Reply:
Good catch - ensured the cookie is removed regardless of domain in 57748ee1706fdd71316a835576e753da41a0b51f